src/system.h

```diff
@@ -208,8 +208,12 @@ struct System {
 
     void LoadTut() {
         auto lang = frame->app->locale.GetCanonicalName();
-        lang.Truncate(2);
-        if (LoadDB(frame->GetDocPath(L"examples/tutorial-" + lang + ".cts"))[0]) {
+
+        if (lang.Len() == 5 && LoadDB(frame->GetDocPath(L"examples/tutorial-" + lang + ".cts"))[0]) {
+            lang.Truncate(2);
+        }
+
+        if (lang.Len() == 2 && LoadDB(frame->GetDocPath(L"examples/tutorial-" + lang + ".cts"))[0]) {

```

Comment:
I guess the difference is the old code would work even if `lang` was empty etc. Recommend making this code more robust by doing something along these lines:
```
if (len == 5 && !LoadDB..) return;  // Success with 5
truncate(2);
if (len == 2 && !LoadDB..) return;  // Success with 2
LoadDB..  // Load default.
```

Reply:
Done.